apps/client/src/app/pages/portfolio/activities/import-activities-dialog/import-activities-dialog.scss

```diff
@@ -32,4 +32,35 @@
     right: 1.5rem;
     top: calc(50% - 10px);
   }
+
+  .custom-button {
+    border: 3px dashed rgba(var(--palette-primary-500), 1);
+    border-radius: 8px;
+    padding: 30px;
+    text-align: center;
```

Comment:
Can you please use the css class `text-center` in the html file

Reply:
Done